cmd/latest.go

```diff
@@ -18,7 +18,7 @@ var latestCmd = &cobra.Command{
 	RunE:  runLatest,
 }
 
-var removePrefix bool = false
+var removePrefix, latestSkipPrereleases bool = false, false
```

Comment:
Use two lines for this, you can create a var group as well. Just creating it on one line looks weird.